src/Transport/Topology/MetaModel/DefaultConnectionStringToNamespaceAliasMapper.cs

```diff
@@ -29,8 +31,9 @@ public EntityAddress Map(EntityAddress value)
             }
 
             var namespaceName = new ConnectionString(value.Suffix).NamespaceName;
-            throw new InvalidOperationException($"Connection string for {namespaceName} hasn't been configured. {Environment.NewLine}" +
+            Logger.Warn($"Connection string for {namespaceName} hasn't been configured. {Environment.NewLine}, replying may not work properly" +
```

Comment:
`ConnectionString.NamespaceName` is a namespace name (e.g. "my-sb-namespace"). Should excepton say that "connection string for namespace names {namespaceName}' was not configured" ?